apps/drilbur/src/websites/backstagepro/filter.ts

```diff
@@ -6,6 +6,8 @@ export type BackstagepoFilter = {
 export enum BackstageproInstrument {
   guitar = 'guitar',
   bass = 'bass',
+  drums = 'drums',
+  vocals = 'vocals_classic',
```

Comment:
oh this might be tricky.... so apparently backstagepro does support multiple types of vocals 
<img width="276" alt="Screenshot 2023-04-29 at 12 53 59" src="https://user-images.githubusercontent.com/18482002/235298956-884fdfbb-94c0-4778-97a0-5d0800e927be.png">
so this either needs to be an array and we merge all results into `vocals` or we have to offer the exact same filter options...

Reply:
I think what would be best is if we would scrape all the vocal types, so implement this as an array and then merge all the posts into a `vocals` category on our side. Therefore we maintain a consistent datastructure. This could be helpful if we want to work on more details, e.g. finding a specific genre through Information Retrieval or so.